sys/include/clif.h

```diff
@@ -0,0 +1,247 @@
+/*
+ * Copyright (C) 2019 HAW Hamburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    sys_clif CoRE Link Format
+ * @ingroup     sys_serialization
+ * @brief       Simple encoding and decoding of CoRE Link Format (RFC 6690)
+ *              strings
+ * @see RFC 6690: https://tools.ietf.org/html/rfc6690
```

Comment:
It would be useful to add a two or three sentence paragraph to describe the high level API (clif_encode_link, clif_decode_link), and the lower level API (other clif functions) on which they are built.